DESCRIPTION

```diff
@@ -12,5 +12,5 @@ URL: https://github.com/JGCRI/gcamwrapper
 Encoding: UTF-8
 LazyData: true
 LinkingTo: Rcpp
-Imports: Rcpp
+Imports: Rcpp, dplyr, yaml, stringr
```

Comment:
In principal yes.  But I don't actually know what the minimum versions are.  And given these packages are used extensively (except Rcpp) and there isn't a way to say version <= (given tidy verse aversion to backwards compatibility) I'm inclined to say we support which ever version and if it doesn't work open an issue and we can try to fix / work around it and if we truly can not then add the minimum.  What do you think?

Reply:
There may be several options for auto-detecting the minimum (e.g., https://stackoverflow.com/questions/38686427/determine-minimum-r-version-for-all-package-dependencies); however, it may be easier to simply state that we only support versions _n_ and greater officially.  For now, let's leave the version ranges off and we can refine a bit after some testing.